lib/ocg_schemas/src/coordinates.rs

```diff
@@ -341,6 +419,13 @@ impl From<AbsBlockPos> for AbsChunkPos {
 // === RelChunkPos
 impl_simple_ivec3_newtype!(RelChunkPos);
 impl_rel_abs_pair!(RelChunkPos, AbsChunkPos);
+
+impl From<AbsChunkPos> for RelChunkPos {
+    fn from(value: AbsChunkPos) -> Self {
+        Self(value.0)
```

Comment:
This is wrong, absolute to relative should be done by subtracting an absolute reference point